pom.xml

```diff
@@ -5151,6 +5151,32 @@
         <version>${version.org.springframework}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-autoconfigure</artifactId>
+        <version>${version.org.springframework.boot}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-configuration-processor</artifactId>
+        <version>${version.org.springframework.boot}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-autoconfigure-processor</artifactId>
+        <version>${version.org.springframework.boot}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-test</artifactId>
+        <version>${version.org.springframework.boot}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-test-autoconfigure</artifactId>
+        <version>${version.org.springframework.boot}</version>
+      </dependency>
```

Comment:
I think we shouldn't replicate dependency management of `spring-boot-dependencies`.

Reply:
Why stop now?  (just kidding)
I agree, we should into reusing their bom, if and only if it only contains spring dependency. So not reuse their platform bom (which also includes hibernate dependencies etc).

But for me, that doesn't block this PR as this one just continues the way we've been doing it so far.

@cuijulian Can you make a jira for trying to reuse the spring bom (the one without hibernate deps etc, if such one exists)?

Reply:
@ge0ffrey The `spring-boot-dependencies` POM definitely manages hibernate dependencies. There should be a way to exclude them when importing. But there may be a Spring BOM without them, I'll have a look. I'll make a Jira, this will be in a separate PR.